src/ScriptValidation.h

```diff
@@ -0,0 +1,77 @@
+// See the file "COPYING" in the main distribution directory for copyright.
+
+#pragma once
+
+#include "zeek/Traverse.h"
+
+namespace zeek::detail
+	{
+
+// Track statements and hooks and validate break and next usage.
+class BreakNextScriptValidation : public TraversalCallback
```

Comment:
I don't think this needs to be in a header file. If we add more validators, that would get a bit awkward. I suggest to move all the code into `ScriptValidation.cc`, and move the function `script_validation()` there as well. Then just add a prototype for `script_validation()` to `ScriptValidation.h` (or maybe some existing header somewhere if a new one feels too much; or even just into `parse.y`).